### ISSUE TITLE
In IE11, when the popper is near the viewport boundary, extra spacing is added to the page creating a scrollbar

### ISSUE THREAD

**Issue by caseyjhol** [FezVrasta/popper.js#636]
https://codepen.io/caseyjhol/pen/BVBOVL

![popper_scroll](https://user-images.githubusercontent.com/2874325/40628674-9e4deffe-6283-11e8-916f-580d0cc403d1.gif)

### Steps to reproduce the problem

1.  Click the ref element

### What is the expected behavior?

The green `pop` element appears above the ref element and the rest of the page remains as is with no scrollbar.

### What went wrong?

Extra spacing is created below the ref element, creating a scrollbar.

This appears to work properly in all browsers except IE10 and IE11.

I'm using a click listener for the popper (and showing it using `display: block` to make the issue easier to see, but it persists without one as well (https://codepen.io/caseyjhol/pen/VdZGRY). This issue is the cause of twbs/bootstrap#24851. I go into more detail here: https://github.com/twbs/bootstrap/issues/24851#issuecomment-392584969.

**Comment by FezVrasta** [comment]
This is not a Popper.js problem, you should apply an initial position to your popper element so that it doesn't cause any overflow (usually top 0 and left 0), we can't do that in Popper.js because it would already be too late.

**Comment by caseyjhol** [comment]
The problem is occurring *because* Popper is applying the style `top: 0, left: 0` to the element. If Popper could detect the element is near the bottom edge of the viewport and set `bottom: 0, left: 0` instead, the issue could be avoided.

**Comment by caseyjhol** [comment]
@FezVrasta I found a way to fix this with a minimal addition of code (just a few lines). Would you be open to a PR?

**Comment by FezVrasta** [comment]
Yes I'd be curious to see your proposed solution, but absolutely it's not a bug, maybe a not super clear behavior. Popper elements are supposed to be positioned (initially, by your CSS) in a way that doesn't cause overflows.

**Comment by caseyjhol** [comment]
The problem occurs because of the `top: 0, left: 0` styling that is applied by Popper prior to being positioned correctly via the transform property. So, even if the Popper is positioned manually beforehand, it's irrelevant, because Popper changes the positioning.

**Comment by craigjennings11** [comment]
I had the same issue, but was able to get around it by disabling `gpuAcceleration`.

```
new Popper(..., ..., {
  modifiers: {
    computeStyle: {
      gpuAcceleration: false,
    },
  },
});
```

**Comment by caseyjhol** [comment]
@craigjennings11 Awesome - looks like that works!
